content/en/docs/reference/glossary/affinity.md

```diff
@@ -0,0 +1,19 @@
+---
+title: affinity
+id: affinity
+date: 2019-01-11
+full_link: /docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/
+short_description: >
+  Node affinity is a set of rules used by the scheduler to determine where a pod can be placed.
```

Comment:
```suggestion
   Rules used by the scheduler to determine where to place pods.
```
1. The revised text is short (and should be).
2. There is more than one kind of affinity.

Reply:
Good to know :)